bricks/presentational/src/statistics-card/styles.shadow.css

```
@@ -50,11 +50,14 @@
 
 :host([outline="border"]) {
   --card-border: 1px solid var(--color-text-divider-line);
+  --card-hovered-border: 1px solid transparent;
+  --card-hovered-box-shadow: var(--connected-overlay-shadow);
   --card-background: var(--antd-card-background);
   --card-border-radius: 6px;
 }
 
 :host([outline="background"]) {
+  --card-hovered-box-shadow: var(--connected-overlay-shadow);
```

没有值也最好补一下，否则如果它的某个上层元素也设置了 `--card-hovered-border` 这里就有问题

```suggestion
:host([outline="background"]) {
  --card-hovered-border: none;
  --card-hovered-box-shadow: var(--connected-overlay-shadow);
```